sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtil.java

```diff
@@ -793,7 +793,7 @@ public void onFailure(GoogleJsonError e, HttpHeaders responseHeaders) throws IOE
 
   public void copy(Iterable<String> srcFilenames, Iterable<String> destFilenames)
       throws IOException {
-    rewriteHelper(srcFilenames, destFilenames, false, false);
+    rewriteHelper(srcFilenames, destFilenames, false, false, false);
```

Comment:
Please add comments to clarify what parameters are (for readability).

Reply:
Done